tensorboard/BUILD

```diff
@@ -91,8 +91,8 @@ py_test(
     deps = [
         ":manager",
         ":version",
-        "//tensorboard/util:tb_logging",
         "//tensorboard:expect_tensorflow_installed",
+        "//tensorboard/util:tb_logging",
```

Comment:
Please revert spurious changes to this file.

If we want to maintain the invariant that are files match Buildifier
style, we’ll add CI tests to make sure that continues to be the case.
Attempting to fix the style on an ongoing basis without backsliding is
hopeless and just pollutes the blame.

Reply:
Done.